### PR TITLE
auto start the CPU profiler

### PR DESCRIPTION
**Pull request by jamestalmage**
This will automatically start the DevTools profiler when running `profile.js`

### CODE REVIEW

File: profile.js [hunk 1]
```diff
@@ -103,4 +104,8 @@ events.on('stats', function () {
 process.argv[2] = JSON.stringify(opts);
 process.argv.length = 3;
 
-require('./lib/test-worker');
+console.profile('AVA test-worker process');
+
+setTimeout(function () {
```
Comment (sindresorhus) [inline profile.js]: Why? And `setImmediate` instead?

Comment (jamestalmage) [inline profile.js]: The profiler seemed to startup async.

**Comment by jamestalmage** [comment]
`setImmediate` works.

`iron-node` works just fine with a synchronous call. With `devtool` doesn't print the "profile started" line unless I use `setImmediate`.

// @mattdesl

**Comment by mattdesl** [comment]
Not sure why there would be a difference. You could try --no-node-timers to see if that is the reason setImmediate was needed.

**Comment by mattdesl** [comment]
I've cloned `ava` and I'm trying to run some tests with devtool.

```sh
devtool node_modules/ava/profile.js <test-file>
```

Are there specific files to use for `<test-file>` that I should be profiling? Or are you just npm linking ava and testing it on random projects?

**Comment by jamestalmage** [comment]
You can link it, or use a fixture

```
devtool profile.js test/fixtures/es2015.js
```

**Comment by mattdesl** [comment]
Pulled from master. Testing with devtool@1.7.6, node@5 and npm@2.14.12, adding the `console.profile/profileEnd` works fine without `setImmediate` for the above fixture. Which test is failing?

**Comment by jamestalmage** [comment]
It doesn't log the  "profiling started" line - I think that may be the only issue. But it makes me paranoid that I might be missing some profiling data

**Comment by sindresorhus** [comment]
@jamestalmage Can you open an issue on `devtool` about this? It's weird that `setImmediate` is required?

**Pull request merged**